Integrations/integration-GoogleVault.yml

```diff
@@ -44,13 +44,18 @@ script:
 
     # If modifying these scopes, delete the file token.json.
     SCOPES = ['https://www.googleapis.com/auth/ediscovery', 'https://www.googleapis.com/auth/devstorage.full_control']
-    DEMISTO_MATTER = "test_search_phishing"
+    DEMISTO_MATTER = 'test_search_phishing'
```

Comment:
why do we have demisto test matter in the code?

Reply:
Removed